src/redux/actions/differentialExpression/loadDifferentialExpression.js

```diff
@@ -26,39 +30,35 @@ const loadDifferentialExpression = (
   };
 
   let pagination = {};
-
   if (tableState) {
     const currentPageSize = tableState.pagination.pageSize;
-
     pagination = {
       orderBy: tableState.sorter.field,
       orderDirection: (tableState.sorter.order === 'ascend') ? 'ASC' : 'DESC',
       offset: ((tableState.pagination.current - 1) * currentPageSize),
       limit: currentPageSize,
       responseKey: 0,
     };
-
+    if (advancedFilters?.length > 0) {
```

Comment:
You can remove the `?` here too, right?

Reply:
I want to keep it there just in case, tests or something else might crash otherwise and this ? wont create problems